roles/bmc_reboot/tasks/main.yml

```diff
@@ -12,3 +12,9 @@
     username: "{{ dpu_bmc_username }}"
     password: "{{ dpu_bmc_password }}"
   delegate_to: localhost
+
+- name: Pause for a bit to allow BMC to Reboot
+  ansible.builtin.pause:
```

Comment:
I think better to track the job of restart here instead of pause.
It would be more better to add a default parameter `bmc_reboot_wait_time` instead of adding `bmc_reboot_delay_seconds` in vars which can track for the job for a specified amount of seconds before exiting.

Reply:
@sachin-apa do you mean using until like this ?
```
- name: Wait until HTTP status is 200
  uri:
    url: 'http://example.org/iportal/'
    return_content: yes
    validate_certs: no
    status_code:
      - 200
  until: uri_output.status == 200
  retries: 24 # Retries for 24 * 5 seconds = 120 seconds = 2 minutes
  delay: 5 # Every 5 seconds
  register: uri_output
  ```

Reply:
@glimchb  we are doing a graceful restart of a bmc, I'm thinking BMC graceful restart would create a job in the BMC and we can track the job instead of the portal.

Reply:
@sachin-apa if BMC is restarting - how we can track the job... maybe you refer to CPU graceful restart, not BMC itself... no ?

Reply:
started to talk to DMTF @mraineri

Reply:
Mike opened https://github.com/ansible-collections/community.general/issues/8051